docker/dev-builder/android/Dockerfile

```diff
@@ -14,11 +14,7 @@ RUN apt-get update && apt-get install -y \
     git \
     build-essential \
     pkg-config \
-    python3 \
-    python3-dev \
-    python3-pip \
-    && pip3 install --upgrade pip \
-    && pip3 install pyarrow
+    python3
```

Comment:
not sure this python3 better drop or not, need disscuss

Reply:
I believe we can remove this